RPS.js

```diff
@@ -29,24 +29,22 @@ function rockChoice () {
         winnerOfRound = 'Draw, there is no winner';
         console.log(winnerOfRound);
         cpuButtonHighlight(cpuRock);
-        playerCPUDraw(playerScore, cpuScore); 
-        return changeBannerText(winnerOfRound);
+        changeBannerText(winnerOfRound);
     }
     else if (playerChoice == 'rock' && cpuChoice === 'paper') {
         winnerOfRound = 'You lose! Paper beats rock!';
         console.log(winnerOfRound);
         cpuButtonHighlight(cpuPaper);
-        cpuIncrementScore(cpuScore, cpuScoreElement);
```

Comment:
these are global variables, so they don't need to be passed in as they're accessed directly.